apps/framework-cli/src/cli/local_webserver.rs

```diff
@@ -337,13 +363,16 @@ async fn ingest_route(
     }
 }
 
+#[allow(clippy::too_many_arguments)]
 async fn router(
     req: Request<hyper::body::Incoming>,
     current_version: String,
     route_table: &RwLock<HashMap<PathBuf, RouteMeta>>,
+    consumption_apis: &RwLock<HashSet<String>>,
     configured_producer: ConfiguredProducer,
     console_config: ConsoleConfig,
```

Comment:
Did you rebase? The console config should be gone and that might enable you to not have to add the clippy argument